docs/OrderUsage.md

```diff
@@ -0,0 +1,176 @@
Platform\\\",\\\"totalPrice\\\":{\\\"type\\\":\\\"centPrecision\\\",\\\"currencyCode\\\":\\\"EUR\\\",\\\"centAmount\\\":18000,\\\"fractionDigits\\\":2},\\\"custom\\\":{\\\"type\\\":{\\\"typeId\\\":\\\"type\\\",\\\"id\\\":\\\"08c2d736-f2d8-4869-a2ef-fd8f56ff48a3\\\"},\\\"fields\\\":{\\\"subscriptionKey\\\":\\\"275c10d0-1f1e-4463-9c7f-66a6abc7ce7b_subscriptionKey\\\",\\\"reminderDays\\\":5,\\\"schedule\\\":\\\"0 0 1 Feb,May,Aug,Nov *\\\",\\\"cutoffDays\\\":5,\\\"isSubscription\\\":true}},\\\"lineItemMode\\\":\\\"Standard\\\"}]: Request body does not contain valid JSON.\",\"name\":\"VError\",\"stack\":\"VError: Unexpected error on creating template order from checkout order with number: 1646302276551. Line item: [{\\\"id\\\":\\\"1bea71c8-7712-4db4-945f-095085e0e69b\\\",\\\"productId\\\":\\\"8671cacd-500e-44ac-b6ee-2beada092441\\\",\\\"productKey\\\":\\\"product-1\\\",\\\"name\\\":{\\\"en\\\":\\\"Wine subscription\\\"},\\\"productType\\\":{\\\"typeId\\\":\\\"product-type\\\",\\\"id\\\":\\\"93b85ce4-bd0a-4c42-9aa5-a45c71915c4a\\\"},\\\"productSlug\\\":{\\\"en\\\":\\\"wine-subscription\\\"},\\\"variant\\\":{\\\"id\\\":1,\\\"sku\\\":\\\"wine01\\\",\\\"prices\\\":[{\\\"id\\\":\\\"91cc2648-5b51-497d-8a24-69b77b762a23\\\",\\\"value\\\":{\\\"type\\\":\\\"centPrecision\\\",\\\"currencyCode\\\":\\\"EUR\\\",\\\"centAmount\\\":18000,\\\"fractionDigits\\\":2},\\\"country\\\":\\\"DE\\\"}],\\\"images\\\":[],\\\"attributes\\\":[],\\\"assets\\\":[]},\\\"price\\\":{\\\"id\\\":\\\"a9c9dd05-ea46-4b34-b6a2-c31592156117\\\",\\\"value\\\":{\\\"type\\\":\\\"centPrecision\\\",\\\"currencyCode\\\":\\\"EUR\\\",\\\"centAmount\\\":18000,\\\"fractionDigits\\\":2},\\\"country\\\":\\\"DE\\\"},\\\"quantity\\\":1,\\\"discountedPricePerQuantity\\\":[],\\\"addedAt\\\":\\\"2022-03-03T10:11:16.817Z\\\",\\\"lastModifiedAt\\\":\\\"2022-03-03T10:11:16.817Z\\\",\\\"state\\\":[{\\\"quantity\\\":1,\\\"state\\\":{\\\"typeId\\\":\\\"state\\\",\\\"id\\\":\\\"e380a2fb-77f3-482b-bf83-2060876200ba\\\"}}],\\\"priceMode\\\":\\\"Platform\\\",\\\"totalPrice\\\":{\\\"type\\\":\\\"centPrecision\\\",\\\"currencyCode\\\":\\\"EUR\\\",\\\"centAmount\\\":18000,\\\"fractionDigits\\\":2},\\\"custom\\\":{\\\"type\\\":{\\\"typeId\\\":\\\"type\\\",\\\"id\\\":\\\"08c2d736-f2d8-4869-a2ef-fd8f56ff48a3\\\"},\\\"fields\\\":{\\\"subscriptionKey\\\":\\\"275c10d0-1f1e-4463-9c7f-66a6abc7ce7b_subscriptionKey\\\",\\\"reminderDays\\\":5,\\\"schedule\\\":\\\"0 0 1 Feb,May,Aug,Nov *\\\",\\\"cutoffDays\\\":5,\\\"isSubscription\\\":true}},\\\"lineItemMode\\\":\\\"Standard\\\"}]: Request body does not contain valid JSON.\\n    at _createTemplateOrderAndPayments (file:///Users/aoz/commercetools-subscriptions/src/create-template-orders.js:182:13)\\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\\n    at async pMap.concurrency (file:///Users/aoz/commercetools-subscriptions/src/create-template-orders.js:79:9)\\n    at async file:///Users/aoz/commercetools-subscriptions/node_modules/p-map/index.js:100:20\"}",
```

Comment:
We might need to anonymize some info in this log in our code. WDYT?

Reply:
🤔 Which information that we need to anonymize ?

Reply:
Not in your PR, but maybe in the code. Can you create a follow-up ticket to anonymize some data when logging error? E.g. email etc.

Reply:
Done: PSS-453